src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/controller/AppointmentsControllerTest.kt

```diff
@@ -109,4 +112,29 @@ internal class AppointmentsControllerTest {
     assertThat(appointmentsResponse.size).isEqualTo(1)
     assertThat(appointmentsResponse.first()).isEqualTo(ActionPlanAppointmentDTO.from(actionPlanAppointment))
   }
+
+  @Test
+  fun `updates appointment with attendance details`() {
```

Comment:
💭 Should we mention some sort of assertion in the test title, about returning the now "attendance recorded" appointment?